examples/rest-crud/src/__tests__/helpers.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-rest-crud
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {EntityCrudRepository} from '@loopback/repository';
+import {Todo} from '../models';
+
+/**
+ * Generate a complete Todo object for use with tests.
+ * @param todo - A partial (or complete) Todo object.
+ */
+export function givenTodo(todo?: Partial<Todo>) {
+  const data = Object.assign(
+    {
+      title: 'do a thing',
+      desc: 'There are some things that need doing',
+      isComplete: false,
+    },
+    todo,
+  );
+  return new Todo(data);
+}
+
+export type TodoRepository = EntityCrudRepository<
```

Comment:
I thought we are generating repos and controllers...why do we need a repo here?

Reply:
It's just a type alias for the tests, not an actual repository, but I'll put a comment.